lib/inventory_refresh/save_collection/saver/concurrent_safe_batch.rb

```diff
@@ -234,6 +236,18 @@ def update_or_destroy_records!(records_batch_iterator, inventory_objects_index,
         records_for_destroy = [] # Cleanup so GC can release it sooner
       end
 
+      def changed?(record, hash, all_attribute_keys)
```

Comment:
This looks really similar to https://github.com/ManageIQ/inventory_refresh/pull/36/files#diff-cb1e66c18054d4336b8cc88b25e66dd5R37 could these be combined?

Reply:
hm, so these are different, one is per record other is per IC

Reply:
:+1: